src/test/java/htsjdk/samtools/util/HistogramTest.java

```diff
@@ -5,17 +5,18 @@
 import org.testng.annotations.Test;
 
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashSet;
 
 import static java.lang.Math.abs;
+import static java.lang.StrictMath.pow;
 
-/**
- *
- */
 public class HistogramTest {
 
```

Comment:
As far as serialization tests go, I think any one of the test ones in this file (or [these](https://github.com/broadinstitute/gatk/blob/4864c3d5fe7ea78245ea03904b27a2ecbb95e5f5/src/test/java/org/broadinstitute/hellbender/tools/spark/sv/InsertSizeMetricsCollectorSparkUnitTest.java#L35)) would do, since I don't think any of them serialized properly before. As long as we get back a proper treemap and bin and value labels.